firstvoices/backend/tasks/import_job_tasks.py

```diff
@@ -113,38 +114,24 @@ def import_resource(
         site=import_job_instance.site,
         importjob=import_job_instance,
         new_rows=result.totals["new"],
-        skipped_rows=result.totals["skip"],
-        error_rows=result.totals["error"] + result.totals["invalid"],
+        skipped_rows=0,
```

Comment:
Do we want to leave a hard-coded variable here ? or we can just remove it from the model, serializer and other places and add it back later when we have a use for it ?

Reply:
Let's remove it for now